src/index.js

```diff
@@ -0,0 +1,83 @@
+import TodoList from './todoList';
+import Todo from './todo';
+import Projects from './projects';
+import Project from './project';
+
+const projects = new Projects();
+projects.listProjects();
+const todoList = new TodoList();
+todoList.listTodos();
+
+function clearFields() {
+  const inputs = document.querySelectorAll('input');
+  inputs.forEach(input => {
+    input.value = '';
+  });
+}
+
+function createTodo(event) {
+  event.preventDefault();
+  const project = document.querySelector('.project');
+  const title = document.querySelector('.todo-title');
+  const description = document.querySelector('.description');
+  const priority = document.querySelector('.priority');
+  const dueDate = document.querySelector('.due-date');
+  const todo = new Todo(project.value, title.value, description.value,
+    dueDate.value, priority.value, false);
+  todo.createCard();
```

Comment:
## Fix todo creation hitch on new projects.
- [ ] A user is unable to create their todos on newly created projects saved for the `default` project as shown below 👇 
![to-do-list](https://user-images.githubusercontent.com/22345961/103947078-9426d480-5148-11eb-9464-e460d489aa4c.png)
. Kindly fix this bug.